Sources/PointFree/Bootstrap.swift

```diff
@@ -9,119 +9,22 @@ import PointFreePrelude
 import PointFreeRouter
 import PostgresKit
 import Prelude
-
-public func bootstrap(eventLoopGroup: EventLoopGroup) -> EitherIO<Error, Prelude.Unit> {
+ 
+public func bootstrap() -> EitherIO<Error, Prelude.Unit> {
```

Comment:
pretty much all of bootstrap goes away since most of this work is in preparing dependencies, and that work just happens in each respective `liveValue`.

Reply:
maybe we don't even need this anymore...